GoogleTestAdapter/DiaResolver/PeParser.cs

```diff
@@ -104,17 +104,26 @@ private static class NativeMethods
             // ReSharper disable once InconsistentNaming
             public const ushort IMAGE_DIRECTORY_ENTRY_IMPORT = 1;
 
-            [DllImport("imageHlp.dll", CallingConvention = CallingConvention.Winapi, CharSet = CharSet.Ansi, BestFitMapping = false, ThrowOnUnmappableChar = true)]
+            [DllImport("imageHlp.dll", CallingConvention = CallingConvention.Winapi, CharSet = CharSet.Ansi, BestFitMapping = false, ThrowOnUnmappableChar = true, SetLastError = true)]
```

Comment:
Here (and below) I'm not sure (note that I don't have too much experience with calling Windows API). Shouldn't we be doing anything with the last error in case one occurs? It looks like we ask the marshaller to grab a possible error, but then ignore the result... What do I miss?

Reply:
Ok, I see - thanks for the explanation! But could we make use of last error to provide more meaningful error messages (in fact, why haven't we done this from the very beginning)? We already have a [utility method](https://github.com/csoltenborn/GoogleTestAdapter/blob/caac8ae382846eb87f66cd6893e89408ac26b2ec/GoogleTestAdapter/Common/Win32Utils.cs#L8) for getting a readable error string...

Reply:
Yes.  To clean the code, I think WinAPI functions should be wrapped in a throwing version that gets the error so that, for example:
```C#
int exitCode;
if(!GetExitCodeProcess(process, out exitCode))
    throw new Win32Exception(Marshal.GetLastWin32Error(), "Could not get exit code of process");

return exitCode;
```
Becomes:
```C#
return GetExitCodeProcess(process);
```

Reply:
Sounds good again... would you mind to tackle this? 

I'm wondering whether it might be slightly cleaner to move the error handling into a different PR and restrict this one to the (awesome) test improvements you are contributing. However, I'm basically fine with both - if you decide to also do the error handling, feel free to choose whatever suites you better.

One more thing: I'm eager to merge #231 (the other result of our bug hunt), but this will conflict with this PR, I assume, so I haven't done that yet to not make your life more difficult. Just keep this in mind, please ;-) (e.g. by not refactoring parts of the laucher/executor/debugger attacher stuff - don't worry about what you have already done, though).

Reply:
Didn't hear anything from you any more - did I ask for too much?

Note that I'm on vacation now for about 2 weeks...